front/src/apollo/apolloProvider.ts

```diff
@@ -1,13 +1,39 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
-
+import { ApolloClient, InMemoryCache, HttpLink, split, makeVar } from '@apollo/client';
+import { getMainDefinition } from '@apollo/client/utilities';
+import { WebSocketLink } from '@apollo/client/link/ws';
```

Comment:
```suggestion
import { ApolloClient, InMemoryCache, HttpLink, split, makeVar, getMainDefinition, WebSocketLink } from '@apollo/client';
```

같은 패키지면 할 수 있을것같은데 이렇게는 임포트가 안되나요? (단순 호기심입니다!)

Reply:
해봤는데 안됩니다. `Module '"@apollo/client"' has no exported member 'WebSocketLink'.ts(2305)` 같은 에러 메시지가 발생합니다.